pagopa/pn-frontend
# PG - accept delegation - pn-1825

## Short description
Now PG is able to accept the delegations from other users (PF or another PG)

## List of changes proposed in this pull request
- Added AcceptanceModal
- Updated AcceptButton component
- Wrote tests

## How to test
Login with a PG that have delegation from other users -> clicks on Acept button -> check that delegations are accepted with and without groups
